src/types/NavigationType.tsx

```diff
@@ -1,3 +1,4 @@
 export interface NavigationType {
   navigate: (path: string, option?: { screen: string, params: object }) => {},
+  dangerouslyGetParent: () => {setOptions({ tabBarVisible })},
```

Comment:
Est-ce que tu as essayé de suivre ce qui était dit sur cette page : https://reactnavigation.org/docs/hiding-tabbar-in-screens ?

Reply:
done

Reply:
tu peux l'enlever du coup non ?